cpp/src/io/comp/debrotli.cu

```diff
@@ -154,25 +156,34 @@ struct debrotli_state_s {
   debrotli_huff_tree_group_s *distance_hgroup;
   uint16_t *block_type_vlc[3];
   huff_scratch_s hs;
-  uint32_t mtf[65];
-  uint64_t heap[LOCAL_HEAP_SIZE / 8];
+  uint8_t mtf[65 * sizeof(uint32_t)];
+  char heap[LOCAL_HEAP_SIZE];
```

Comment:
since the original is unsigned, do we want to keep it unsigned?
```suggestion
  uint8_t heap[LOCAL_HEAP_SIZE];
```

Reply:
@jrhemstad recommended to use `char` here. AFAIK the type is not that important, as we are only using this as a memory pool.  The :allocated" chunks are returned as void*.